packages/core/core-flows/src/cart/workflows/update-line-item-in-cart.ts

```diff
@@ -49,7 +49,7 @@ export const updateLineItemInCartWorkflow = createWorkflow(
     validateVariantPricesStep({ variants })
 
     const items = transform({ input }, (data) => {
-      return [data.input.item]
+      return [{...data.input.item, quantity: data.input.update.quantity as number }]
```

Comment:
suggestion: since the workflow already works with copy, would it make sense to instead use Object.assign in order to not do a shallow copy that might not be needed here? it would look like `Object.assign(data.input.item, { quantity: ... })`